src/stars_star.css

```diff
@@ -0,0 +1,7 @@
+.stars_star {
+  background-image: url(images/star-grey.png);
```

Comment:
Again, it's better to use the svg given by the project. The .png does not work on the tests